packages/backend-storage/src/construct.ts

```diff
@@ -32,6 +29,11 @@ const storageStackType = 'storage-S3';
 export type AmplifyStorageTriggerEvent = 'onDelete' | 'onUpload';
 
 export type AmplifyStorageProps = {
+  /**
+   * Whether this storage resource is the default storage resource for the backend
+   * @default false
```

Comment:
isn't the default true? ie if I just have one storage instance and I don't define this field, then it's the default

Reply:
Good point. I don't know if the default should be `true` or `false` because it's conditional. If there's only one bucket, then it's `true`. If multiple buckets, it's `false`. Maybe I should update the JSdoc to clarify the conditions

Reply:
Done 
bc4d412